email-service/src/main/java/org/java/mentorship/email/config/ApiConfig.java

```diff
@@ -0,0 +1,18 @@
+package org.java.mentorship.email.config;
+
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.Primary;
+import org.springframework.ui.freemarker.FreeMarkerConfigurationFactoryBean;
+
+@Configuration
+public class ApiConfig {
+
+    @Primary
```

Comment:
Does it need the Primary annotation?
Primary is used when there are multiple Beans of this type and it makes sure spring chooses the one annotated with Primary

Reply:
I believe it does. Since removing the Primary annotation mess with running the EmailApplication.

![image](https://github.com/user-attachments/assets/bf414e17-3964-4b54-bd6a-37a6d189febb)